app/components/live-sidebar.hbs

```diff
@@ -18,7 +18,7 @@
   <div data-test-sidebar-body class='sidebar__body {{if this.isSideBarOpen "open" "close"}}'>
     <div data-test-sidebar-body-host class='body__host'>
       <span data-test-sidebar-host-container class='host__container'>
-        <img data-test-sidebar-host-image class='host__image' src={{@hostProfilePicture}} alt='host' />
+        <img data-test-sidebar-host-image class='host__image' src={{if @hostProfilePicture @hostProfilePicture 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png' }} alt='host' />
```

Comment:
Can we please use this image to make it consistent on all sites https://github.com/Real-Dev-Squad/website-www/blob/develop/img/profile.png